templates/includes/sidebar.html

```diff
@@ -1,4 +1,13 @@
 <ul class="sidebar-nav">
```

Comment:
Make sure you've added a blank line at the end of all your files, including .html files, as per the checklist.